realtime/presence.py

```diff
@@ -2,18 +2,70 @@
  Defines the RealtimePresence class and its dependencies.
 """
 
-from typing import Any, Callable, Dict, List, Optional
+from enum import StrEnum
```

Comment:
`enum.StrEnum` was added in Python `3.11.0`, [see this link last line of that paragraph.](https://devdocs.io/python~3.12/library/enum#enum.StrEnum)

Before that you gotta use something like:

```python
import enum

class MyEnum(enum.Enum, str):
  KEY0 = "VALUE0"
  KEY1 = "VALUE1"

```